src/megaapi_impl.cpp

```diff
@@ -17372,6 +17397,7 @@ MegaTransferDataPrivate::MegaTransferDataPrivate(const MegaTransferDataPrivate *
     this->uploadTags = transferData->uploadTags;
     this->downloadPriorities = transferData->downloadPriorities;
     this->uploadPriorities = transferData->uploadPriorities;
+    this->notificationNumber = transferData->notificationNumber;
```

Comment:
I have made it that way to keep it consistent with the lines above. Since it should be a bit more efficient and I can't see any strong reason to change all those assignments now, I think that we can continue with that approach there for now.